OtripleS.Portal.Web/.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
I guess this was not intentionally added

Reply:
true story, t'm using vs code on mac